app/assets/javascripts/govuk_publishing_components/analytics/page-content.js

```diff
@@ -14,7 +14,7 @@
       case isDocumentCollectionPage():
         return document.querySelectorAll('.document-collection .group-title').length
       case isMainstreamBrowsePage():
-        return countVisible(document.querySelectorAll('#subsection ul')) || document.querySelectorAll('#section ul').length
+        return countVisible(document.querySelectorAll('#subsection ul')) || document.querySelectorAll('#section ul').length || document.querySelectorAll('#root ul').length
```

Comment:
It only counts the `#section ul` and `#root ul` as having 1 section each... I guess that makes sense (looking at https://www.gov.uk/browse/births-deaths-marriages/marriage-divorce) as they don't have sections in the way that the `#subsection ul` part has. Just wanted to check that was expected behaviour? 

Also if it is expanded enough to see the `'#subsection ul'` then it only counts the subsections within that menu... again, is this expected behaviour rather than it adding the total of all 3 menus (say if it were for https://www.gov.uk/browse/births-deaths-marriages/marriage-divorce)?

Reply:
If so then it looks good to me!

Reply:
Yes, this is expected behaviour. Good to ask though, as it is a bit confusing. `root` and `section` only ever have one list, which is why counting is a bit simpler, whereas `subsection` can have many lists, hence the extra effort of the `countVisible` function to calculate it.

And yes, we only want to count the column that is visible on the right. So if it can't count the `subsection`, it tries to count the `section` and if it can't find that it counts the `root`.